Neos.Neos/Classes/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -190,6 +190,10 @@ protected function setPropertyLabels($nodeTypeName, array &$configuration)
                 $propertyConfiguration['ui']['aloha']['placeholder'] = $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, 'aloha.placeholder');
             }
 
+            if (isset($propertyConfiguration['ui']['inline']['editorOptions']) && $this->shouldFetchTranslation($propertyConfiguration['ui']['inline']['editorOptions'], 'placeholder')) {
+                $propertyConfiguration['ui']['inline']['editorOptions']['placeholder'] = $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, 'inline.placeholder');
```

Comment:
Shouldn't it be `'ui.inline.editorOptions.placeholder'`? /cc @kitsunet @kdambekalns what do you think?